debug.log

```diff
@@ -0,0 +1,2 @@
+[0103/194400.592:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[0103/210407.672:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Šis failas čia nereikalingas, reiktų arba jį ištrinti arba įdėti i .gitignore

Reply:
my bad, nemačiau 👍